pkg/downloader/manager.go

```diff
@@ -89,6 +89,10 @@ func (m *Manager) Build() error {
 		return err
 	}
 
+	if c.Metadata.Dependencies == nil && m.Debug {
+		fmt.Print("No dependencies found to build")
```

Comment:
This call is inside the Helm SDK and is used other applications. We can't assume it's a CLI here.

In these cases we use...

```go
fmt.Fprintln(m.Out, "No dependencies found to build")
```

This will send the output to the right location whether it's the CLI or some other location an SDK user needs it.